mybatis.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Rules to check SQL statements in MyBatis Mapper XML files.
 homepageUrl=https://github.com/donhui/sonar-mybatis
-archivedVersions=1.0.1,1.0.2,1.0.3
-publicVersions=1.0.4
+archivedVersions=1.0.1,1.0.2,1.0.3,1.0.4
+publicVersions=1.0.5
 
 defaults.mavenGroupId=org.sonarsource.plugins.mybatis
 defaults.mavenArtifactId=sonar-mybatis-plugin
 
+1.0.5.description=bugfix: handle typeHandler,javaType,jdbcType,parameterMap,cache,cache-ref when generate reduced.xml and update mybatis version to 3.5.2
+1.0.5.sqVersions=[6.7,LATEST]
```

Comment:
6.7 is End-of-Life. Please update this to start at 7.9

Reply:
> start at 7.9

Thanks for your suggestion, I have updated it.